matter_server/client/model/mappable_device.py

```diff
@@ -0,0 +1,79 @@
+"""Helper class to help split a node in mappable devices."""
+from __future__ import annotations
+from abc import ABC, abstractmethod
+
+from typing import TYPE_CHECKING
+
+
+from matter_server.vendor import device_types
+from matter_server.vendor.chip.clusters import Objects as all_clusters
+
+
+if TYPE_CHECKING:
+    from .node import MatterNode
+    from .device import MatterDevice
+
+
+def mappable_devices_from_node(node: MatterNode) -> list[MappableDevice]:
+    """Split a node in mappable devices."""
+    if not node.bridge_device:
+        return [MappableNodeDevice(node)]
+
+    devices = []
+
+    for device in node.devices:
+        if device.device_type == device_types.BridgedDevice:
```

Comment:
Shouldn't that be `!=`?

Reply:
Hm, from what I can tell we exclude the `BridgedDevice` anyways when building `devices` list not?

Reply:
we exclude `Bridge`, not `BridgedDevice` . We use `BridgedDevice` to indicate which endpoints are their own devices.